netlify/open-api
# docs: update dependency redoc-cli

This PR updates the [redoc-cli](https://www.npmjs.com/package/redoc-cli) dependency to upgrade the look and feel of our API reference docs. It also sorts the operation in the sidebar alphabetically, closing #388. 

## Test plan

1. Navigate to the [Deploy Preview](https://deploy-preview-390--open-api.netlify.app/) and ensure nothing seems buggy or out of place
2. Ensure that clicking on an operation in the sidebar adds a hash to the URL, and refreshing the page takes you to the same anchor
